.github/workflows/android.yml

```diff
@@ -0,0 +1,48 @@
+name: Build APK
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: 'zulu'
+
+      - name: Build Debug APK with Gradle
+        run: chmod +x ./gradlew && ./gradlew assembleDebug
+
+      - name: Upload apk as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: app
+          path: app/build/outputs/apk/debug/app-debug.apk
+
+      - name: Push build
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && github.repository == 'X1nto/OpenCord'
```

Comment:
this line isn't really necessary

Reply:
I just makes sure this isn't running unless it's the master branch, not a PR, and not a fork before trying to send it over webhook, why is that not necessary

Reply:
I meant the push action itself since APK will mostly be larger than 8MB, which won't allow us to send the file to Discord. What we could do is instead send the link to the artifact to Discord

Reply:
I'm not sure a good way to do that, so I guess you can just manually send it for now